### PR TITLE
chore: enable typescript

### PR DESCRIPTION
**Pull request by gasp**
enable developers to use typescript for new services

### CODE REVIEW

**Review by tpombeiro87 — APPROVED** [review]
👏

**Comment by gasp** [comment]
inspired by
* maze-api migrate to serverless https://github.com/mazedesignhq/maze-api/commit/530da4ce452219209d7945882a12de316cfa8bae
* babel + ts PR https://github.com/postlight/serverless-babel-starter/pull/42

File: package.json [hunk 1]
```diff
@@ -8,10 +8,10 @@
     "deploy": "export NODE_ENV=development && yarn deploy:env",
     "deploy:production": "export NODE_ENV=production && yarn deploy:env",
     "deploy:stage": "export NODE_ENV=stage && yarn deploy:env",
-    "lint": "eslint src --ext js --no-ignore --fix",
+    "lint": "tslint -c tslint.json --fix './src/**/*.ts' && eslint src --ext js --no-ignore --fix",
```
Comment (tgroutars) [inline package.json]: Please don't use `tslint` as it's being shut down in favor of `eslint`

File: jest.config.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ module.exports = {
   testRegex: '(.*\\.test\\.(tsx?|jsx?))$',
   transform: {
     '^.+\\.jsx?$': 'babel-jest',
+    '^.+\\.tsx?$': 'ts-jest',
```
Comment (tgroutars) [inline jest.config.js]: `jest` supports TypeScript via babel so we shouldn't need `ts-jest`

**Review by tgroutars — CHANGES_REQUESTED** [review]

**Review by tgroutars — APPROVED** [review]
🙌

**Pull request merged**